frontend/src/routes/SearchPage/components/SearchResults.tsx

```diff
@@ -87,7 +87,7 @@ function RenderRelatedTables(
                             keyFn={(item: any) => item._uid.toString()}
                             tableActions={[]}
                             rowActions={
-                                kind !== 'cluster' && kind !== 'release'
+                                kind !== 'cluster' && kind !== 'release' && kind !== 'policyreport'
```

Comment:
Maybe it's better if we control this by updating the RBAC role and rolebinding?

This works in the UI only and I could still cause problems with `oc delete policyreport ...`.
We don't want the UI to drift from the CLI.  The other 2 cases here cluster and release aren't actual kube resources.

Reply:
@jlpadilla Here is the PR to add view auth for all users: https://github.com/open-cluster-management/multiclusterhub-operator/pull/586